parse-changelog.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+cd "$(dirname $0)"
+
+docker run --rm \
+  -v $PWD/..:/src/conjur-server \
```

Comment:
When I run this script locally in the cyberark/summon-s3 tree, it can't find the CHANGELOG.md file.
I think this line needs to be:
```
  -v $PWD:/src/conjur-server \
```